### PR TITLE
added sidebar,avatar

### PR DESCRIPTION
**Pull request by Asuraking1n**

### CODE REVIEW

**Comment by rishichourasia** [comment]
All things seems good , 
1) just that width of the sidebar looks way more than required , maybe 15% would look good enough. 
2) your nav logo is shiting to left side on decreasing screen size . small fix there 
other looks good to me, that loading gif is amazing

File: css/docs.css [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+.doc-cont{
+    width: 100%;
+    height: 87vh;
+    background-color: #fff;
+    display: flex;
+}
+
+.doc-sidebar{
+    width: 20%;
```
Comment (rishichourasia) [inline css/docs.css:17]: maybe try 13%-16% which ever looks good enough

Comment (Asuraking1n) [reply css/docs.css:17]: Sure I will do it.

**Comment by Balakumar3018** [comment]
All looks good. 
1st iframe div aligned in left It would be better if you change second i frame code div to full left as first one.

**Pull request merged**